src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -474,11 +475,21 @@ export default class OncoprintControls extends React.Component<
         ) {
             return _.map(
                 this.props.state.heatmapProfilesPromise.result,
-                profile => ({
-                    label: profile.name,
-                    value: profile.molecularProfileId,
-                    type: profile.molecularAlterationType,
-                })
+                profile => {
+                    let label = profile.name;
+                    if (
+                        profile.molecularAlterationType ===
+                        AlterationTypeConstants.MUTATION_EXTENDED
+                    ) {
+                        label = `Variant Allele Frequency`;
```

Comment:
Hardcoding `Variant Allele Frequency` might be problematic in the future if we want to add a new mutation related track other than VAF. Not sure what's the best way to address this issue though.